docs/adr/0016-auth-for-settings-redesign.md

```diff
@@ -0,0 +1,114 @@
+# Authentication in Settings Redesign
```

Comment:
Since we made a decision, and now we're making a new decision, we probably want a new title here - e.g. Switch from OAuth2 to unique Bearer token in settings redesign auth.